tools/python/python

```diff
@@ -0,0 +1 @@
+/usr/bin/python
```

Comment:
Your GIT somehow added this file which is installed from the get.py script.  This needs to be removed.

Reply:
Same comment about `placeholder_for_Arduino` addition

Reply:
These have been removed from the latest commit.